### ISSUE TITLE
Editorial issues (mostly consistency) within the WCAG2ICT language

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#618]
Will use this to gather any editorial changes needed that I found:

- [x] 1.2.1: the note should say "success criterion" instead of "success criteria".
- [x] 1.3.1: Note 1 - Change "through the use of accessibility services provided by platform software" to "by using the accessibility services of platform software". This will use the exact verbiage used in the defined key term.
- [x] 1.4.10 Reflow: Note 7, 3rd paragraph - add a link to the "platform software" key term.
- [ ] 1.4.11 Non-text contrast: 2 places - word substitution of "user agent or platform software" - maybe should be "user agent or other platform software". **MJM: To be fixed using Issue #612**
- [ ] 1.4.13 Content on Hover or Focus: 3 places - word substitution of "user agent or platform software" - maybe should be "user agent or other platform software" (also in Note 1) **MJM: To be fixed using Issue #612**
- [x] 2.2.2: Note 2 - The second replacement should be "in the non-web document or software" instead of "non-web documents and software" and then modify the statement that talks about the word replacements to match. **MJM: Being fixed using issue #680**
- [x] 2.3.1: Note 1 - Similar to 2.2.2 where the replacement should be "in the non-web document or software" to make the sentence read better. Also, "the whole software" should be replaced by "the whole software application" or "the whole software program" to be clearer. **MJM: Being fixed using issue #680**
- [x] 2.5.1 Pointer gestures, 2.5.2 Pointer Cancellation, and 2.5.7 Dragging Movements - word substitutions and Note - "non-web software application" - change to "non-web software" and "non-web software applications that interpret" to "non-web software that interprets" to be consistent with the EN 301 549
- [x] 2.5.3 Page Titled - Note 3 that points to closed functionality section needs "(ADDED)" notation. **MJM: Was fixed before we got to this issue**
- [ ] 2.5.7: SC verbiage, 2 places.  Should the word replacement be "user agent or other platform software" rather than "user agent or platform software"?  I think our definition of platform software gives a user agent as an example. **MJM: To be fixed using Issue #612**
- [ ] 2.5.8 Target size (minimum), 3 places - Similar to 2.5.7 we have word replacement of "user agent or platform software". Should it instead be "User agent or other platform software"? **MJM: To be fixed using Issue #612**
- [x] 3.1.1 Language of page - Note 1 should replace "comply with" and "meet" to "satisfy" to be consistent with verbiage used elsewhere.
- [x] 4.1.2 Name, role, value - Note 2 "accessibility services provided by platform software" should be "accessibility services of platform software" and link to that key term rather than the platform software key term.
- [ ] 4.1.3 Status messages - Similar verbiage used "user agent or platform software" - maybe should be "user agent or other platform software".  **MJM: To be fixed using Issue #612**
- [x] "large scale (text)" glossary term, 3 places - word substitution and Note 5: "browsers, user agents, or platform software” should probably be "browsers, user agents, or other platform software”. But check that the definition of platform software includes browsers as an example of platform software.
- [x] "style property" glossary term, 5 places - Might need updating...depending on what examples are provided for platform software - is a user agent platform software? Word substitution of "user agent(s)" with "user agent(s) or platform software" and "user agent interface settings" with "user agent, platform software, or other software interface settings, or"  - add "other" before platform software to be "user agent(s) or other platform software in 2 places. 
- [x] "large scale (text)" - Need to remove Note 2 which no longer exists in WCAG.

**Comment by ChrisLoiselle** [comment]
Hi @maryjom is this the result of https://github.com/w3c/wcag2ict/issues/580 or something different? If different, are we planning to bring this up in a meeting or update per editorial only? Thanks for clarifying.

**Comment by maryjom** [comment]
@ChrisLoiselle This issue came about when I was doing a review of the EN 301 549 and I noticed these anomalies in our guidance where there are some inconsistencies in language.
